### PR TITLE
[GH-920] Consolidate wgs modules

### PR DESCRIPTION
**Pull request by rexwangcc**
### Purpose

- https://broadinstitute.atlassian.net/browse/GH-920

### Changes

- Consolidate `wl.clj` and `wgs.clj`
- Fix an error handling issue with `/api/v1/workload`.
- Test the result `wgs.clj` and fix some potential issues with the env lookup.

### Review Instructions

- **Note: this PR should NOT be merged before #127 since there are merge conflicts that this PR is supposed to resolve.**
- **Note: this PR REMOVES the CLI part of the WGS module, which means it is a breaking change.**
- Run `clojure -A:test` locally and make sure the test passes.

### CODE REVIEW

File: api/src/zero/module/wgs.clj [hunk 1]
```diff
@@ -204,18 +184,89 @@
...
+                        env (str input input_cram) output)))])
+            (update! [tx [id uuid]]
+              (when uuid
+                (jdbc/update! tx items
+                  {:updated now :uuid uuid}
+                  ["id = ?" id])))]
+      (let [workflows (postgres/get-table tx items)
+            ids-uuids (map submit! workflows)]
+        (jdbc/update! tx :workload {:started now} ["uuid = ?" uuid])
+        (run! (partial update! tx) ids-uuids)))))
```
Comment (rexwangcc) [inline api/src/zero/module/wgs.clj:272]:
```suggestion
        (run! (partial update! tx) ids-uuids)))))

```

File: api/test/zero/tools/workloads.clj [hunk 2]
```diff
@@ -39,6 +39,7 @@
   "An aou arrays sample for testing."
   {:cromwell      "https://cromwell-gotc-auth.gotc-dev.broadinstitute.org/",
    :notifications [{:chip_well_barcode           "7991775143_R01C01",
+                    :call_rate_threshold         0.98
```
Comment (samanehsan) [inline api/test/zero/tools/workloads.clj:42]: Ah! I just added this in my last PR as well (it's on line 45) so you can remove this one

**Review by samanehsan — APPROVED** [review]
LGTM! :octocat:

File: api/test/zero/tools/workloads.clj (hunk 2 above)
Comment (rexwangcc) [reply api/test/zero/tools/workloads.clj]: Oops, I just noticed it!

**Pull request merged**